File: socket_test.py
from socket_ import SocketSender
import time
ss = SocketSender(10000, "192.168.0.103")

message = ['text', {'content_list': ['You come la, sensei.', 'Still is old sheep son', 'look get up one face tired status ya.'], 'emotion': 'smile'}]

ss.send(message)

time.sleep(2)

change = ['size_final', 2]

ss.send(change)

ss.close()

File: socket_test.js
const net = require('net');

const client = new net.Socket();
client.connect(10000, '192.168.0.104', () => {
    console.log('Connected');
    const message = JSON.stringify(['text', {'content_list': ['Hello from Node.js!'], 'emotion': 'smile'}]);
    client.write(message);
    client.destroy();
});